notebook/static/notebook/js/quickhelp.js

```diff
@@ -127,7 +127,7 @@ define([
     
     function humanize_key(key){
         if (key.length === 1){
-            return key.toUpperCase();
+            return key;
```

Comment:
If we aren't casting to upper case, we should probably cast to lower case. It's probably important that it be consistent.